pkg/token/athenz.go

```diff
@@ -87,6 +89,7 @@ func fetchAccessToken(ztsClient *zts.ZTSClient, t CacheKey, saService string) (*
 		role:   t.Role,
 		raw:    accessTokenResponse.Access_token,
 		expiry: int64(*accessTokenResponse.Expires_in),
```

Comment:
The `expiry` field should be timestamp instead of the TTL.
The `expires_in` field should be computed as `now - at.expiry` OR the API definition should be changed. (sidecar does NOT follow the spec description)

sidetrack:

https://github.com/AthenZ/athenz/blob/044d78244667e864e0a21708faab61145879267b/servers/zts/src/main/java/com/yahoo/athenz/zts/ZTSImpl.java#L2582C29-L2582C50
`access token expiry_in` should be identical to role token `maxExpiryTime`. Can you also help to change the related parts?